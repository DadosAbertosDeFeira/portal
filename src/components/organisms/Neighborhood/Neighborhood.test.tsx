import { render } from "@testing-library/react";

import { Neighborhood, type NeighborhoodProps } from ".";

describe("Neighborhood", () => {
  const supportersMock = ["One", "Two", "Three", "Four"];

  const makeSut = ({
    items = supportersMock,
    ...props
  }: Partial<NeighborhoodProps>) => {
    return render(<Neighborhood items={items} {...props} />);
  };

  it("should renders correctly", () => {
    const { asFragment } = makeSut({});

    expect(asFragment).toMatchSnapshot();
  });
});
