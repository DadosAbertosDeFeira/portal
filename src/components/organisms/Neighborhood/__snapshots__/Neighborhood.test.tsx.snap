// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Neighborhood should renders correctly 1`] = `[Function]`;
